src/components/common/Layout/components/Account/components/AccountModal/components/LinkAddressButton.tsx

```diff
@@ -40,6 +40,7 @@ const LinkAddressButton = (props) => {
 
     try {
       await walletClient.requestPermissions({ eth_accounts: {} })
+    } catch {
```

Comment:
Just out of curiosity, does this change have an effect, or just left here by accident? 😄 (I'm actually not really sure, if the `catch` is missing, the error gets caught or not :harold:)

Reply:
I added the catch because the error didn't get caught haha (and it's a completely valid case that the user just clicks on "Cancel" inside the MetaMask popup - well, that caused a pretty ugly error :harold:)

Reply:
Wow, nice to know it works like that, would have guessed, that it still catches, just doesn't do anything with it 🤔 I guess it makes sense since a `somePromise.then(...).finally(...)` would also reject
Thanks, I learned something today 😄